met-web/src/components/engagement/form/AddSurveyBlock.tsx

```diff
@@ -95,6 +135,20 @@ export const AddSurveyBlock = () => {
                     spacing={2}
                     sx={{ padding: '1em' }}
                 >
+                    <Grid item xs={12}>
+                        <Box sx={{ width: '100%', typography: 'body1' }}>
+                            <MetTabs tabs={tabs} updateTabName={updateTabName} />
+                        </Box>
+                        <If condition={selectedTabName == SubmissionStatus[SubmissionStatus.Upcoming]}>
+                            <Then> {richtexteditor(headerUpcoming, paragraphUpcoming)} </Then>
+                        </If>
+                        <If condition={selectedTabName == SubmissionStatus[SubmissionStatus.Open]}>
+                            <Then> {richtexteditor(headerUpcoming, paragraphOpen)} </Then>
```

Comment:
functions that return JSX and React Nodes and should called like this /<RichTextEditor header={headerUpcoming" paragraph={paragraphOpen} /> instead